CONTRIBUTING.md

```diff
@@ -77,7 +77,7 @@ Here's how to run the test suite:
   creates (a default AWS or GCE disk size won't work -- see
   [#6243](https://github.com/go-gitea/gitea/issues/6243)).
 - Change into the base directory of your copy of the gitea repository,
-  and run `drone exec --local --build-event pull_request`.
+  and run `drone exec --event pull_request`.
```

Comment:
This line is unrelated to this PR, but as it is small rather than open a second PR for just this I have included here. If there is opposition to including this line in this PR please comment here and I will open a second PR for this line.

Reply:
I think there is an open issue related to that. I will look into it.

Reply:
It was closed by an other PR but this one complete it #7350